Sources/Latte+UniversalLink.swift

```diff
@@ -60,6 +106,46 @@ public extension Latte {
             return nil
         }
     }
+
+    static func getUniversalLinkHandler(
+        userActivity: NSUserActivity,
+        universalLinkOrigin: URL,
+        rewriteUniversalLinkOrigin: URL? = nil
+    ) -> LatteLinkHandler? {
+        guard userActivity.activityType == NSUserActivityTypeBrowsingWeb,
+              let incomingURL = userActivity.webpageURL else {
+            return nil
+        }
+
+        return getUniversalLinkHandler(
+            incomingURL: incomingURL,
+            universalLinkOrigin: universalLinkOrigin,
+            rewriteUniversalLinkOrigin: rewriteUniversalLinkOrigin
+        )
+    }
+
+    static func getShortLinkExpander(
```

Comment:
I suggest we do not expose unnecessary object (i.e. the Expander) in the API. What about we expose a `static func expand`?

Reply:
@louischan-oursky I designed the api in this way because:
1. We need an object to be the `URLSessionDelegate`.
2. We need a function which returns synchronously therefore we can know if the userActivity is able to be handled by the short link expander. If no, the userActivity should be passed to the next handler (Which is probably `getUniversalLinkHandler`). If there is only a `expand` function, as expanding the url requires calling api, it cannot be synchronous.